docs/sql-performance-tuning.md

```diff
@@ -228,6 +228,8 @@ The "REPARTITION_BY_RANGE" hint must have column names and a partition number is
     SELECT /*+ REPARTITION */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(c) */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(3, c) */ * FROM t
+    SELECT /*+ ADAPTIVE_REPARTITION */ * FROM t
+    SELECT /*+ ADAPTIVE_REPARTITION(c) */ * FROM t
 
 For more details please refer to the documentation of [Partitioning Hints](sql-ref-syntax-qry-select-hints.html#partitioning-hints).
```

Comment:
I think we need to add more details about the new hint in `sql-ref-syntax-qry-select-hints`.

Reply:
There are other hints that seems AQE can work optimize, could we have more doc about the difference between each under AQE?

Reply:
yea, it's better to add docs for the new hint. Since this PR only adds a new hint that has no big difference with previous hint, we can add more docs and use case after adding new rule.